Automattic/wp-calypso
# Move paragraph block out of wpcom-site-editor

#### Changes proposed in this Pull Request

* As https://github.com/Automattic/wp-calypso/pull/58741#issuecomment-994743696 mentioned, I misunderstand the meaning of the `wpcom-site-editor` before. Thus, creating the new paragraph-block under the ETK plugin to customize the setting of the paragraph block such as placeholder.

#### Testing instructions



- Ensure your API and site are pointed to the sandbox
- Open another terminal and run cd apps/editing-toolkit && yarn dev --sync
- Go to the editor
- Check the placeholder of title, body and paragraph are correct!



Related to https://github.com/Automattic/wp-calypso/pull/58741#issuecomment-994743696
